src/lib/Microsoft.Health.Fhir.Ingest.Common/Data/IMeasurement.cs

```diff
@@ -1,29 +1,29 @@
-﻿// -------------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
-// -------------------------------------------------------------------------------------------------
-
-using System;
-using System.Collections.Generic;
-
-namespace Microsoft.Health.Fhir.Ingest.Data

```

Comment:
Should we update the namespace in all of these files? 
Microsoft.Health.Fhir.Ingest.Common.Data

Reply:
I don't think so.  We don't need to align libraries with the namespaces.  This is a refactor to split out the libraries.  I would suggest the library should be call Microsoft.Health.Fhir.Ingest.Schema or Microsoft.Health.Fhir.Ingest.Contracts since that is what is being stored in the library.

Reply:
The namespace will remain same in this refactoring work to avoid other changes on the consumer classes.

For the assembly name, I think "Schema" suggested by Dustin would sound better. I'll update it.